main/zfs-vanilla/APKBUILD

```diff
@@ -3,8 +3,8 @@
 
 # when changing _ver we *must* bump _rel
 _name=zfs
-_ver=0.8.1
-_rel=0
+_ver=0.8.2
+_rel=1
```

Comment:
shouldn't it be 0 ?

Reply:
How would users be upgraded to the new version then?